### DISCUSSION TITLE
Assistance in Linking HA entities with Loxone Virtual Inputs

### DISCUSSION THREAD

**Discussion by DeepflashX** [JoDehli/PyLoxone#349] [Q&A]
Hello all together,

can someone please support me? I am coming from Loxone and want to add more integrations into it, which is why I installed HA on a Raspberry Pi and PyLoxone.
It is working but I don`t know, how to link existing HA entities (Like Grohe Guard Temperature for example) with a Loxone Virtual Input.
Can someone guide me?

Best regards

**Comment by JoDehli** [comment]
@DeepflashX do you want to use ha as your main platform or loxone?

**Comment by DeepflashX** [comment]
I would like to use Loxone as main platform.
HA-lox works well but the other way round I have no clue. I would love to link a Loxone Switch with a switch in HA. All is present as entities in HA.

**Reply by JoDehli** [reply]
@DeepflashX Pyloxone is not good for sending data to loxone. If you want to send data to loxone I would use node red. Controlling loxone from Homeassistent is the preferred way to use Pyloxone.

**Comment by DeepflashX** [comment]
@JoDehli yes I also have Node Red but want to merge Node Red and Home Assistant. Home Assistant has more Integrations but Node Red works better with Loxone.. But the sending to Loxone is not the problem, I do have a Blueprint with which it works great, just the other way round is a problem.

**Comment by JoDehli** [comment]
@DeepflashX I thought you want to link ha entities with loxone

**Comment by DeepflashX** [comment]
Yes to change something in Home Assistant.
I want to link a switch in Loxone to a switch in Home Assistant so when I change the switch in Loxone, the switch in HA will do the same.
or sync an analog value from Loxone to HA so when I change it in Loxone it will be changed accordingly in HA.
Sending analog values from HA to Loxone works well.

**Reply by JoDehli** [reply]
@DeepflashX I really do see the point. You can make a switch in loxone and you can control it in Homeassistant. And what you control with this switch can be done in Homeassistent using a script or automation. You can even sync it with another switch.

**Reply by DeepflashX** [reply]
I am really having a hard time figuring out, how to change something within HA from Loxone.
For example: I have the Grohe Integration and there is a water valve which can be turned on and off.
Now I want to control that valve from within Loxone.
How?  :)
